Source/Events/EventSourceId.cs

```diff
@@ -9,24 +9,24 @@ namespace Dolittle.SDK.Events
     /// <summary>
     /// Represents the identification of an event source.
     /// </summary>
-    public class EventSourceId : ConceptAs<Guid>
+    public class EventSourceId : ConceptAs<string>
     {
         /// <summary>
         /// Implicitly convert from a <see cref="Guid"/> to an <see cref="EventSourceId"/>.
         /// </summary>
         /// <param name="eventSourceId">EventSourceId as <see cref="Guid"/>.</param>
-        public static implicit operator EventSourceId(Guid eventSourceId) => new EventSourceId { Value = eventSourceId };
+        public static implicit operator EventSourceId(Guid eventSourceId) => new EventSourceId { Value = eventSourceId.ToString() };
```

Comment:
Same as I mentioned for PartitionId

Reply:
Let's leave this for now until we get the results back from the poll.